scripts/preprocessing/04b-run_ssp.py

```diff
@@ -21,6 +23,30 @@
 
 
 @failsafe_run(on_error=on_error)
+
+def _get_global_reject_ssp(raw):
+    if 'eog' in raw:
+        eog_epochs = mne.preprocessing.create_eog_epochs(raw)
+    else:
+        eog_epochs = []
+    if len(eog_epochs) >= 5:
+        reject_eog = get_rejection_threshold(eog_epochs, decim=8)
+        del reject_eog['eog']  # we don't want to reject eog based on eog
+    else:
+        reject_eog = None
+
+    ecg_epochs = mne.preprocessing.create_ecg_epochs(raw)
+    # we will always have an ECG as long as there are magnetometers
+    if len(ecg_epochs) >= 5:
+        reject_ecg = get_rejection_threshold(ecg_epochs, decim=8)
+        # here we want the eog
+    else:
+        reject_ecg = None
+
+    if reject_eog is None and reject_ecg is not None:
+        reject_eog = {k: v for k, v in reject_ecg.items() if k != 'eog'}
```

Comment:
```suggestion
    if reject_eog is None and reject_ecg is not None:
        reject_eog = dict(reject_ecg)
        if 'eog' in reject_eog:
             del reject_eog['eog']
```

it seems that's what it does. But no idea why.... @dengemann can you help?

Reply:
EOG will remove too many trials. It's removed by ICA in a later step

Reply:
Yes, precisely as @jasmainak says. The goal is to estimate EOG artefacts, this is why you do not want to remove them before the SSP estimation. The 1-liner was much more readable though to me ... but apparently only to me.